gemini-k8s-cluster

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+#
+# Convenient script to run gemini on cluster
+#
+# N.B:
+# make sure kubectl has the same version as server
+
+# variables
+declare -r DB_HOST='scylladb'
+declare -r SPARK_MASTER='spark://p-spark-master:7077'
+
+# create pod if not exists
+if ! kubectl get pods | grep gemini > /dev/null; then
+  kubectl create -f ./k8s/gemini-pod.yaml
+fi
+
+while ! kubectl get pods | grep "gemini.*Running" > /dev/null; do
+  sleep 1
+done
+
+case $1 in
+  hash)
+    kubectl exec gemini -- /bin/bash -c "MASTER=$SPARK_MASTER /gemini/hash -h $DB_HOST ${@:2}"
```

Comment:
If the `80chars` limit is a hard limit, there are some lines that should be modified.

Reply:
I think in this particular case it's better to exceed the limit.